draft-ietf-anima-autonomic-control-plane/draft-ietf-anima-autonomic-control-plane.xml

```diff
@@ -1198,31 +1198,44 @@ will be distributed with the flooded objective.  An example of the message is in
 
                                 <t>The following sub-sections define the security association protocols that are considered to be important and feasible to specify in this document.</t>
 
-                        <section anchor="IPsec-group" title="ACP via IKEv2">
+                        <section anchor="IPsec-group" title="ACP via IPsec">
 
-                        <t>An ACP node announces its ability to support IKEv2 as the ACP secure channel protocol in GRASP as "IKEv2".</t>
+<!--kaduk I did not propose to change the GRASP method name here, since I do not know how disruptive that would be.  But the ACP data channel is carried over IPsec, not IKEv2, so from that POV changing the method name would be recommended.-->
+                        <t>An ACP node announces its ability to support IPsec, negotiated via IKEv2, as the ACP secure channel protocol in GRASP as the "IKEv2" method for the "AN_ACP" objective.</t>
 
                             <section anchor="IPsec" toc="include" title="Native IPsec">
 
                                 <t>To run ACP via IPsec natively, no further IANA assignments/definitions are required.</t>
 
-<t> An ACP node that is supporting native IPsec MUST use IPsec security setup via IKEv2 for tunnel mode and IPsec/IKE signaling accordingly for IPv6 payload (e.g.: ESP next header of 41). It  MUST use local and peer link-local IPv6 addresses for encapsulation.</t>
+<t>The ACP usage of IPsec and IKEv2 mandates a narrow profile of the current standards-track usage guidance for IPsec <xref target="RFC8221"/> and IKEv2 <xref target="RFC8247"/>.  This profile provides for stringent security properties and can exclude deprecated/legacy algorithms because there is no need for interoperability with legacy equipment for ACP channels.  Any such backward compatibility would lead only to increased attack surface and implementation complexity, for no benefit.</t>
 
-<t>Authentication MUST use the ACP domain certificates. Certificate Encoding MUST support "PKCS #7 wrapped X.509 certificate" (0) (see <xref target="IKEV2IANA"/> for this and other IANA IKEv2 parameter names used in this text). If certificate chains are used, all intermediate certificates up to, but not including the locally provisioned trust anchor certificate must be signaled.</t>
+<t> An ACP node that is supporting native IPsec MUST use IPsec in tunnel mode, negotiated via IKEv2, and with IPv6 payload (e.g., ESP Next Header of 41). It MUST use local and peer link-local IPv6 addresses for encapsulation.  Manual keying MUST NOT be used (independent of the security weaknesses of manual keying, it is incompatible with an autonomic platform).</t>
```

Comment:
(IIUC this text as-written does not exclude using AH without ESP, i.e., using IPsec without encryption.  Mandating ESP would be the way to require encryption, if desired.)

Reply:
You have written a bunch of IPsec requirements, replacing the text that was a bunch of IKEv2 requirements. We want to say, MUST use IKEv2.

Reply:
The IKEv2 requirements were basically moved, not removed -- I tried to keep the IPsec bits and the IKEv2 bits grouped together.